### DISCUSSION TITLE
Does Open3D has a touch support?

### DISCUSSION THREAD

**Discussion by ezorfa** [isl-org/Open3D#3742] [Q&A] [answered]
Hello @ssheorey  !

I was wondering if Open3D has a touch support to take touch inputs for non-blocking 3d visualizations. 

Thanks a lot!

**Comment by ssheorey** [comment]
Open3D itself does not interpret touch or gestures. Touchscreens are usually interpreted as a mouse by the OS and thus there is indirect support. Were you looking for anything specific like gestures or multi-touch support? They are not supported.

**Comment by ezorfa** [comment]
@ssheorey Thanks for your reply! My use case is as follows: I am planning to use Open3D in Jetson Nano with a display device which is basically a touch screen. I would like to visualize 3D models and interact with it. Is such a scenario feasable?

**Comment by ssheorey** [comment, marked as answer]
Based on some testing on a Linux touchscreen laptop, basic clicking and dragging works, so this should work for you too.
